src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -36,10 +36,9 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, string indexDir
                 string accountName = m.Groups["accountname"].Value;
                 string containerName = m.Groups["containername"].Value;
                 string relativePath = m.Groups["relativepath"].Value;
-                string feedUrl = m.Groups["feedurl"].Value;
 
                 bool isPublic = string.IsNullOrWhiteSpace(m.Groups["token"].Value);
-                this.feed = new BlobFeed(accountName, accountKey, containerName, feedUrl, string.IsNullOrWhiteSpace(indexDirectory) ? Path.GetTempPath() : indexDirectory, Log, isPublic);
+                this.feed = new BlobFeed(accountName, accountKey, containerName, relativePath, string.IsNullOrWhiteSpace(indexDirectory) ? Path.GetTempPath() : indexDirectory, Log, isPublic);
```

Comment:
I don't think this is going to work. We need should use the exact full feedUrl not a relativePath.

Reply:
BlobFeed.RelativePath is set by this, which calculates the relativeUrl (which is the part of the url after the feed.blob.core.windows.net/containername). It seems to work from my testing, what makes you suspect otherwise?

Reply:
Addressed the concern of a different url, by writing feedUrl into the index.json.